osu.Game/Graphics/Containers/OsuFocusedOverlayContainer.cs

```diff
@@ -28,7 +28,7 @@ public abstract class OsuFocusedOverlayContainer : FocusedOverlayContainer, IPre
         protected override bool BlockNonPositionalInput => true;
 
         /// <summary>
-        /// Temporary to allow for overlays in the main screen content to not dim theirselves.
+        /// Temporary to allow for overlays in the main screen content to not dim themselves.
```

Comment:
Huge nit, but if we're fixing English - this `for` there is majorly throwing me off. This seems better?

```suggestion
        /// Temporary to allow overlays in the main screen content to not dim themselves.
```

Reply:
99% sure this is correct english (and describes the thing better in my head).

Reply:
Agree with the used english.